corehq/apps/app_manager/static/app_manager/js/settings/translations.js

```diff
@@ -1,12 +1,13 @@
 hqDefine("app_manager/js/settings/translations", function () {
     var appTranslationsModel = function (options) {
-        hqImport("hqwebapp/js/assert_properties").assertRequired(options, ['baseUrl', 'lang']);
+        hqImport("hqwebapp/js/assert_properties").assertRequired(options, ['baseUrl', 'lang', 'skipBlacklisted']);
```

Comment:
since the approach has changed to a broader translations and not just blacklisted, the naming would need to adapt to that.
How about `onlyTransifexEligible`?
Similarly in the url, `skipbl` would need to change.